e2e/loadgen/app/config.go

```diff
@@ -0,0 +1,71 @@
+package app
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/lib/evmchain"
+	"github.com/omni-network/omni/lib/netconf"
+)
+
+type Config struct {
+	Count        int
+	Network      string
+	Role         string
+	ChainIDPairs string
+}
+
+type ChainIDPair struct {
```

Comment:
It is better to work with ChainIDs, ie uint64s, not chain names, so lets change this to from string to uint64.